src/styles/animations.css

```diff
@@ -29,3 +29,33 @@
 	animation-fill-mode: both;
 	animation-name: fadeOut;
 }
+
+@keyframes slideInDown {
+	from {
+		max-height: 0;
+	}
+	to {
+		max-height: 300px;
+	}
+}
+
+.slideInDown {
```

Comment:
These were originally sliding animations, which personally I think is a nice effect, but collapse/expand seems more common for accordions, and in any case, what we currently have is collapse/expand. Renamed to `expandDown` and `collapseUp`.